packaging/pom.xml

```diff
@@ -231,7 +231,7 @@
                 <postCp>server/target/lib/*.jar</postCp>
               </classPath>
               <jre>
-                <path>%JAVA_HOME%;%PATH%</path>
+                <path>server/target/jre</path>
```

Comment:
We should be able to produce two different Windows packages: one with embedded JRE, one without embedded JRE (for people who already have Java on their machine). So instead of modifying the existing Launch4j configuration, we would need to duplicate it, to be able to run with both `<path>%JAVA_HOME%;%PATH%</path>` and with `<path>server/target/jre</path>`. Perhaps you could take inspiration from the profiles introduced by @afkbrb in #2498?

Reply:
Hi @wetneb thanks for the suggestion. I looked the up the issue #2498 , i understand one has to use profiles to generate two different types of exec depending on the option given during compilation. I will update my commit with the suggested inclusion.